core/silkworm/consensus/base/engine.cpp

```diff
@@ -68,7 +69,8 @@ ValidationResult ConsensusEngineBase::pre_validate_block(const silkworm::Block&
             return ValidationResult::kNotAnOmmer;
         }
 
-        if (as_range::any_of(old_ommers, [&ommer](const BlockHeader& old_ommer) -> bool { return old_ommer == ommer; })) {
+        if (as_range::any_of(old_ommers,
+                             [&ommer](const BlockHeader& old_ommer) -> bool { return old_ommer == ommer; })) {
```

Comment:
I think:
```
if (as_range::find(old_ommers, ommer) != old_ommers.end()) {
```
is clearer as to the intent

Reply:
Nice! Done in https://github.com/torquem-ch/silkworm/pull/492/commits/149f6666bccf05b3d17aca753650780082624d71